### DISCUSSION TITLE
Move website to clean-css.github.io

### DISCUSSION THREAD

**Discussion by johannchopin** [jakubpawlowicz/clean-css#1174] [Ideas]
Since the repo is now moved to his own organisation, we could extract the `/website` folder to his own repo in the organisation called `clean-css.github.io` so it can be deployed on https://clean-css.github.io instead of https://clean-css.github.io/clean-css/.

**Comment by jakubpawlowicz** [comment]
Makes sense to me :+1: Do you have rights to do so?

**Reply by johannchopin** [reply]
I should have them since I can create a new repo. I will let you know if I face some problems.

**Reply by jakubpawlowicz** [reply]
:+1:
